CRM/Contact/BAO/Query.php

```diff
@@ -5829,12 +5811,40 @@ public function convertToPseudoNames(&$dao, $return = FALSE, $usedForAPI = FALSE
         $val = $dao->{$value['idCol']};
         if ($key == 'groups') {
           $dao->groups = $this->convertGroupIDStringToLabelString($dao, $val);
-          return;
+          continue;
```

Comment:
or at the very leaset maybe a false positive not sure i see it come up in phpcs on php7